src/Common/src/Interop/Unix/libc/Interop.getaddrinfo.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+internal static partial class Interop
+{
+    internal static partial class libc
+    {
+        [DllImport(Libraries.Libc, SetLastError = true)]
```

Comment:
Can you please be explicit about setting CharSet to Ansi here (and for an P/Invoke which marshals strings as UTF-8)?